path_helper.py

```diff
@@ -0,0 +1,13 @@
+import pathlib
+
+
+class PathHelper:
+    @staticmethod
+    def to_absolute(relative: str) -> str:
+        return str(pathlib.Path(relative).resolve())
+
+    @staticmethod
+    def replace_extension(input_path: str, new: str) -> str:
```

Comment:
Looking how you use it above, I can say that it works a bit different than I thought before. It's now operating extensions, but suffixes indeed. So it's safe to call it something like `replace_suffix()`.